### ISSUE TITLE
Multiline pie labels

### ISSUE THREAD

**Issue by DZDomi** [swimlane/ngx-charts#337]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x ] feature request
[ ] support request => Please do not submit support request here
```

Is there any way to make the label of the pie charts span multiple lines? As it is now, the labels will be trimed instead of spanning multiple lines. 

How can i fix this problem (screenshot is of a version where I disabled trimming in the trim-label-helper)?
![image](https://cloud.githubusercontent.com/assets/4184533/25188099/3104b960-2525-11e7-9631-89eac9030717.png)

**Comment by marjan-georgiev** [comment]
There is not one at the moment. The reason for that is that we have an algorithm which positions the labels in a way that they don't overlap, but it assumes they are all of the same height (single line). 

If we allow multi-line labels we would have to modify that algorithm, but there would be no guarantee that overlaps will be prevented, because there would be no limit on the length of the labels.

Another issue here is that the labels also have a limited horizontal space, which is why we truncate them. 

I am open to suggestions on ways to handle these use cases.

**Comment by khaled-ansary** [comment]
Thanks open this issue. It would be great  if display a tooltip with full text or extend the label with multi-lines  when we place mouse (mouse-over event) on the label. Just an alternative suggestion.

**Comment by marjan-georgiev** [comment]
This was added and released in 7.1.0
